app/policies/user_policy.rb

```diff
@@ -0,0 +1,32 @@
+class UserPolicy
+  attr_reader :user, :record
+
+  class Scope
+    attr_reader :user, :record, :scope
+
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    def resolve
+      if user.email.downcase.include?("@digital.cabinet-office.gov.uk")
+        scope
+          .all
+      end
+    end
+  end
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def can_manage_user?
+    # Phase 1
+    user.email.downcase.include?("@digital.cabinet-office.gov.uk")
```

Comment:
Raised a comment on the spike PR but I guess I'm still not entirely sure what the rationale for the phased approach is - couldn't we do phase 2 and add a migration or a script to make our accounts / `@digital.cabinet-office.gov.uk` accounts into super admins?

Reply:
https://github.com/alphagov/forms-admin/pull/372#issuecomment-1497471076